src/utils/fetchClient.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Consider removing the 'eslint-disable' line and use proper typing for your data parameter instead of 'any'. This will ensure better type safety and avoid potential issues.